src/Mvc/Mvc.Core/ref/Microsoft.AspNetCore.Mvc.Core.csproj

```diff
@@ -15,7 +15,6 @@
     <Reference Include="Microsoft.AspNetCore.Routing.Abstractions"  />
     <Reference Include="Microsoft.AspNetCore.Routing"  />
     <Reference Include="Microsoft.Extensions.DependencyInjection"  />
-    <Reference Include="Microsoft.Extensions.DependencyModel"  />
```

Comment:
@natemcmaster we could remove `Microsoft.Extensions.DependencyModel` from the shared fx now. None of the other MVC assemblies that reference it are part of Microsoft.AspNetCore.App. Should I remove it?

Reply:
verified S P I C Y

Reply:
Let's run this by Damian and Fowler. Can you send mail on this? I'm personally okay with it, but let's make sure we think measure the impact first.